src/app/asset-page/licenses.ts

```diff
@@ -32,9 +32,9 @@ export const licenses = [
       img: 'https://licensebuttons.net/l/by-nc-nd/4.0/88x31.png'
     },
     { name: 'Creative Commons: Public Domain Mark',
-      link: 'https://creativecommons.org/share-your-work/public-domain/',
-      img: 'https://licensebuttons.net/p/88x31.png',
-      html: '<a href="https://creativecommons.org/publicdomain/zero/1.0/" title="CC0 1.0 Universal (CC0 1.0) Public Domain Dedication">Public Domain</a></div>'
+      link: 'https://creativecommons.org/publicdomain/mark/1.0/',
+      img: 'https://licensebuttons.net/p/mark/1.0/88x31.png ',
+      html: '<a href="https://creativecommons.org/publicdomain/mark/1.0/" title="CC0 1.0 Universal (CC0 1.0) Public Domain Dedication">Public Domain</a></div>'
```

Comment:
Does this line need the closing div at the end? It seems a little weird to have no opening one but im not sure how this html is injected on the page

Reply:
Discussed offline and its not causing any issues so we won't touch it.